src/main/java/org/liquidengine/legui/system/renderer/nvg/NvgRenderer.java

```diff
@@ -39,12 +36,13 @@
     protected Map<String, Font> loadedFonts = new ConcurrentHashMap<>();
     private long nvgContext;
     private NvgLoadableImageReferenceManager imageReferenceManager;
+    private boolean isVersionNew = (glGetInteger(GL30.GL_MAJOR_VERSION) > 3) || (glGetInteger(GL30.GL_MAJOR_VERSION) == 3 && glGetInteger(GL30.GL_MINOR_VERSION) >= 2);
```

Comment:
Initialization of variable should be created in method `initialize`. 
The main point is that such calls are context related and method `initialize` called in thread with binded context.